apps/migrate/src/app/app.service.ts

```diff
@@ -846,7 +846,7 @@ export class AppService {
       return JSON.parse(correctJsonSyntax);
     } catch (error) {
       console.error('Could not parse:', correctJsonSyntax);
-      throw new Error(error);
+      throw new Error(error as string); // @TODO :Investigate this, whecn just new Error(error)

```

Comment:
I have not looked but do we have a tech debt or regular research ticket written for this?